code/game/objects/structures/lattice.dm

```diff
@@ -67,7 +67,18 @@
 				user << "<span class='notice'>Slicing lattice joints ...</span>"
 			new /obj/item/stack/rods(src.loc)
 			qdel(src)
-
+		return
+	// VOREStation Edit - Added Catwalks

```

Comment:
:eyes:

Reply:
Funny eyes don't explain to me what is wrong. Most of what I did should be a direct copy of how the code is set up on Vorestation.

Reply:
I think the implication is that we don't need the comment there.

Reply:
Mostly just a joke. It's a comment added on VORE to show where the code is edited on said server, so that doing syncs with here is easier. It's just slightly amusing in that aspect. It's completely unnecessary but doesn't really do anything.